config/manifests/bases/opendatahub-operator.clusterserviceversion.yaml

```diff
@@ -2,31 +2,319 @@ apiVersion: operators.coreos.com/v1alpha1
 kind: ClusterServiceVersion
 metadata:
   annotations:
-    alm-examples: '[]'
-    capabilities: Basic Install
-  name: opendatahub-operator.v0.0.0
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "kfdef.apps.kubeflow.org/v1",
+          "kind": "KfDef",
+          "metadata": {
+            "name": "opendatahub",
+            "namespace": "opendatahub"
+          },
+          "spec": {
+            "applications": [
+              {
+                "kustomizeConfig": {
+                  "repoRef": {
+                    "name": "manifests",
+                    "path": "odh-common"
+                  }
+                },
+                "name": "odh-common"
+              },
+              {
+                "kustomizeConfig": {
+                  "repoRef": {
+                    "name": "manifests",
+                    "path": "odh-dashboard"
+                  }
+                },
+                "name": "odh-dashboard"
+              },
+              {
+                "kustomizeConfig": {
```

Comment:
/hold

Sorry missed this earlier, are we moving away from grafana ?

Reply:
Another good catch. I populated this from an odh-core kfdef where I was testing the removal of grafana since the dashboards have not been updated in over a year and are for applications that no longer exist in ODH